src/main/resources/project/procedures/PuppetLint/form.xml

```diff
@@ -0,0 +1,139 @@
+<!--
+
+     Copyright 2015 Electric Cloud, Inc.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
+<editor>
+    <help>/commander/pages/@PLUGIN_NAME@/@PLUGIN_KEY@_help?s=Administration&amp;ss=Plugins#RunManifest</help>
+    <formElement>
+        <type>entry</type>
+        <label>File Path:</label>
+        <value></value>
+        <property>file_path</property>
+        <required>1</required>
+        <documentation>Provide the path to the file for linting. (Required) </documentation>
```

Comment:
There are three input fields for files/paths. To avoid confusion for the user, lets make the labels and their tooltips as clear as possible.

1. File Path -> Manifest file to test [US: I am assuming one can only check manifest files]
2. Config file -> PuppetLint Configuration File
3. Checks file -> What is the exact file this represents in puppet?
Use similar language in all the tooltips. Something like "Absolute path of the xyz file..."